src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -808,6 +814,8 @@ void goto_analyzer_parse_optionst::help()
     " --unreachable-functions      list functions unreachable from the entry point\n"
     // NOLINTNEXTLINE(whitespace/line_length)
     " --reachable-functions        list functions reachable from the entry point\n"
+    // NOLINTNEXTLINE(whitespace/line_length)
+    " --show-callsites filename    list function call graph as CSV (includes functions for pointers, '-' for stdout)\n"
```

Comment:
It's different in (1) it does not print CBMC functions that do not occur in the actual source, and (2) it prints the list of all functions whose address is consumed, so that I now that I have to treat those functions specially when running follow up analysis.

I do not understand why that should be moved into goto-instrument. The code displays something about the goto binary. I do not want to implement the follow up analysis in CPROVER, as that can be done Python. I just want to make relevant parts of the goto-program consumable for Python. If that should be moved to goto-instrument, I can do it.